Libraries/LibJS/Tests/functions/arrow-functions.js

```diff
@@ -90,6 +90,8 @@ test("currying", () => {
 
 test("with comma operator", () => {
     let foo, bar;
+    // prettier-ignore
+    // Ignored so the expressions arent wrapped in parens
     (foo = bar), baz => {};
```

Comment:
We prefer our commits to be atomic. So if the purpose of this commit is to remove files from .prettierignore, then the changes to handle differences from prettier 3.6 should be in another commit. Personally, I don't mind if prettier adds parentheses here so long as older prettier versions (namely, the version used in CI) don't complain. We also have the option to update the version of prettier used in CI.

Reply:
Thank you for the feedback. I assumed the current CI version couldn't handle the 'using' declaration, because of the comment in `.prettierignore`. 

Luckily I was incorrect and when using the CI version `3.3.3`, it doesn't add these extra parentheses. So I will remove these `// prettier-ignore` spots and with `3.3.3` the code remains the same.

If we want to upgrade later we can do the upgrade in CI and update the changes to be more atomic like you suggest.